Epoch 0:
	train loss:1088641.8960  val ap:0.710006  val auc:0.750250
	total time:28.68s sample time:0.12s prep time:18.70s
Epoch 1:
	train loss:899768.2649  val ap:0.718327  val auc:0.764743
	total time:27.61s sample time:0.12s prep time:18.57s
Loading model at epoch 0...
Traceback (most recent call last):
  File "/raid/guorui/workspace/dgnn/a-tgl/train.py", line 246, in <module>
    model.load_state_dict(torch.load(path_saver))
  File "/raid/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/raid/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/raid/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'models/LASTFM_1721917233.6766834.pkl'
